src/editor/Collaboration.js

```diff
@@ -12,14 +12,22 @@ define(function (require, exports, module) {
             // immediately ask for camera access
             autoRequestMedia: false
         });
+        var hash = location.hash.replace(/^#/, "");
+        var m = /&?collaboration=([^&]*)/.exec(hash);
+        if(m && m[1]) {
+            this.room = m[1];
+        } else {
+            this.room = Math.random().toString(36).substring(7);
+        }
+        console.log("Link -> http://localhost:8000/src/hosted.html#?collaboration=" + this.room);
         this.webrtc = webrtc;
         this.pending = []; // pending clients that need to be initialized.
         this.changing = false;
     };
 
     Collaboration.prototype.init = function(codemirror) {
         var self = this;
-        this.webrtc.joinRoom('thimble', function() {
+        this.webrtc.joinRoom("brackets-"+this.room, function() {
```

Comment:
I don't think we need to add the `brackets` prefix here. Let's force the hosting app to enforce uniqueness of rooms (and namespacing if necessary).